packages/flutter_tools/lib/src/cache.dart

```diff
@@ -62,10 +62,11 @@ class Cache {
         locked = true;
       } on FileSystemException {
         if (!printed) {
-          printStatus('Waiting to be able to obtain lock of Flutter binary artifacts directory...');
+          printTrace('Waiting to be able to obtain lock of Flutter binary artifacts directory: ${_lock.path}');
+          printStatus('Waiting for another flutter command to finish...');
```

Comment:
it's not necessarily waiting for the other to finish, that's why I had it phrased the way I had it phrased. :-)